src/Authenticator/LoginHandler.php

```diff
@@ -102,7 +96,9 @@ public function doLogin($data, MemberLoginForm $form, HTTPRequest $request)
         }
 
         // Store a reference to the member in session
-        $this->getStore()->setMember($member)->save($request);
+        /** @var StoreInterface $store */
+        $store = Injector::inst()->create(StoreInterface::class, $member);
+        $store->save($request);
```

Comment:
Just checking that you don't want this store to be persisted, because it's creating an instance of it and not setting it to the handler

Reply:
That's a good point. I do want it persisted - it just so happens that this is the last time the store is accessed. Perhaps to future proof I should ensure it's set on the property here too...

Reply:
Yeah good idea. It would also remove any ambiguity.